app/routes.py

```diff
@@ -1 +1,234 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, make_response, request
```

Comment:
When we have multiple models, our number of routes really starts to add up, so don't be afraid to separate them into multiple files!